snuba/utils/streams/batching.py

```diff
@@ -114,19 +126,18 @@ class BatchingKafkaConsumer:
 
     def __init__(
         self,
-        topics: Union[str, Sequence[str]],
+        consumer: Consumer,
```

Comment:
Will this become an abstract consumer or will the batching consumer always be tied to the confluent kafka consumer ?

Reply:
Nevermind, I did not see the comment in the previous commit.

Reply:
Yeah, it's hard to keep those visible at the top level with the master merges sometimes. ☹️